pkg/deploy/gateway/kube_rbac_proxy.go

```diff
@@ -57,6 +65,7 @@ func getKubeRbacProxyContainerSpec(instance *chev2.CheCluster) corev1.Container
 			"--upstream=http://127.0.0.1:8090/ping",
 			"--logtostderr=true",
 			"--config-file=/etc/kube-rbac-proxy/authorization-config.yaml",
+			"--v=" + strconv.FormatInt(int64(logLevel), 10),
```

Comment:
It is about turning on/off verbose mode?

Reply:
Yes, this is where the verbosity is specified:

kube-rbac-proxy uses `glog` flags for its logging: https://github.com/openshift/kube-rbac-proxy/blob/58e09297223ca912cba8b369bfad150b0fc04a9f/README.md#L13

Example in kube-rbac-proxy deployment:
https://github.com/openshift/kube-rbac-proxy/blob/58e09297223ca912cba8b369bfad150b0fc04a9f/examples/non-resource-url/deployment.yaml#L68

The verbosity is basically an integer: https://pkg.go.dev/github.com/golang/glog#V